entities/weapons/gmod_tool/stools/shareprops.lua

```diff
@@ -39,7 +39,7 @@ function TOOL:LeftClick(trace)
         umsg.Bool(Toolgun)
         if ent.AllowedPlayers then
             umsg.Long(#ent.AllowedPlayers)
-            for k,v in pairs(ent.AllowedPlayers) do
```

Comment:
This is a file of Falco's prop protection. The lint fails on this file, because it should be modified in https://github.com/FPtje/Falcos-Prop-protection instead